doc/modules/model_evaluation.rst

```diff
@@ -1085,8 +1085,30 @@ Regression metrics
 
 The :mod:`sklearn.metrics` module implements several loss, score, and utility
 functions to measure regression performance. Some of those have been enhanced
-to handle the multioutput case: :func:`mean_absolute_error`,
-:func:`mean_squared_error`, :func:`median_absolute_error` and :func:`r2_score`.
+to handle the multioutput case: :func:`mean_squared_error`,
+:func:`mean_absolute_error`, :func:`explained_variance_score` and
+:func:`r2_score`.
+
+
+These functions have an ``multioutput`` keyword argument which specifies
+the way the scores or losses for each individual target should be averaged. The default
+is ``'uniform_average'``, which specifies a uniformly weighted mean over outputs. If
+an ``ndarray`` of shape ``(n_outputs,)`` is passed, then its entries are
+interpreted as weights and an according weighted average is returned. If
+``multioutput`` is ``'raw_values'`` is specified, then all unaltered individual
+scores or losses will be returned in an array of shape ``(n_outputs,)``.
+
+
+The :func:`r2_score` and :func:`explained_variance_score`
+accept an additional value ``'variance_weighted'`` of the option
+``multioutput``, which will lead to a weighting of
```

Comment:
The following seems more correct to me:

"accept an additional value ``'variance_weighted'`` for the ``multioutput`` parameter. This option leads to a weighting..."